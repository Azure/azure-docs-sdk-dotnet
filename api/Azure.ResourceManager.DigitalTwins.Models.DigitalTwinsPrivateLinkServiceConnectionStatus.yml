### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus&text=DigitalTwinsPrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.op_Equality(Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus,Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.op_Inequality(Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus,Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Equals(Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Disconnected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
commentId: T:Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
namespace: Azure.ResourceManager.DigitalTwins.Models
name: DigitalTwinsPrivateLinkServiceConnectionStatus
fullName: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
nameWithType: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.DigitalTwins.dll
packagesWithMoniker:
- value: Azure.ResourceManager.DigitalTwins v1.2.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.DigitalTwins v1.3.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct DigitalTwinsPrivateLinkServiceConnectionStatus : IEquatable<Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type DigitalTwinsPrivateLinkServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure DigitalTwinsPrivateLinkServiceConnectionStatus

      Implements IEquatable(Of DigitalTwinsPrivateLinkServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The status of a private endpoint connection.
metadata:
  api_name:
  - Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.DigitalTwins.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
  - Azure::ResourceManager::DigitalTwins::Models::DigitalTwinsPrivateLinkServiceConnectionStatus
  - DigitalTwinsPrivateLinkServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
