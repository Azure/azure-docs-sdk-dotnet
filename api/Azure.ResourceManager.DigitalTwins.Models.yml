### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.DigitalTwins.Models.ArmDigitalTwinsModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DataExplorerConnectionProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsDescriptionPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsEndpointResourceProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsEventGridProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsEventHubProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsManagedIdentityReference
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsNameContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsNameResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateEndpointConnectionProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkResourceProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsServiceBusProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.TimeSeriesDatabaseConnectionProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.DigitalTwins.Models.CleanupConnectionArtifact
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsAuthenticationType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsEndpointProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsManagedIdentityType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsNameUnavailableReason
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkResourceProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPublicNetworkAccess
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsResourceType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.RecordPropertyAndItemRemoval
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.DigitalTwins.Models.TimeSeriesDatabaseConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.DigitalTwins.Models
commentId: N:Azure.ResourceManager.DigitalTwins.Models
name: Azure.ResourceManager.DigitalTwins.Models
fullName: Azure.ResourceManager.DigitalTwins.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.DigitalTwins.Models
  - Azure::ResourceManager::DigitalTwins::Models
  - Models
  - DigitalTwins.Models
  - DigitalTwins::Models
  helpviewer_keywords:
  - Azure.ResourceManager.DigitalTwins.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
