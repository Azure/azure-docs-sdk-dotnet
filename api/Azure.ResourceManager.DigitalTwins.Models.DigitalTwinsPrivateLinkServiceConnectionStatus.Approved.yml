### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus&text=DigitalTwinsPrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  nameWithType: DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As DigitalTwinsPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.DigitalTwins.Models
name: Approved
fullName: Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
nameWithType: DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.DigitalTwins.dll
packagesWithMoniker:
- value: Azure.ResourceManager.DigitalTwins v1.2.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.DigitalTwins v1.3.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  - Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.DigitalTwins.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  - Azure::ResourceManager::DigitalTwins::Models::DigitalTwinsPrivateLinkServiceConnectionStatus::Approved
  - Approved
  - DigitalTwinsPrivateLinkServiceConnectionStatus.Approved
  - DigitalTwinsPrivateLinkServiceConnectionStatus::Approved
  helpviewer_keywords:
  - DigitalTwinsPrivateLinkServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct DigitalTwinsPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
